test/wallet/signerTest.ts

```diff
@@ -0,0 +1,173 @@
+    const seed = "ss1x3KLrSvfg7irFc1D929WXZ7z9H";
+    const tx3: Transaction = {
+      TransactionType: "Payment",
+      Account: "rHLEki8gPUMnF72JnuALvnAMRhRemzhRke",
+      Amount: "22000000",
+      Destination: "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+      Flags: 2147483648,
+      LastLedgerSequence: 20582339,
+      Fee: "12",
+      Sequence: 20582260,
+    };
+    const signedTxBlob =
+      "120000228000000024013A0F74201B013A0FC36140000000014FB18068400000000000000C732102A8A44DB3D4C73EEEE11DFE54D2029103B776AA8A8D293A91D645977C9DF5F544744730450221009ECB5324717E14DD6970126271F05BC2626D2A8FA9F3797555D417F8257C1E6002206BDD74A0F30425F2BA9DB69C90F21B3E27735C190FB4F3A640F066ACBBF06AD98114B3263BD0A9BF9DFDBBBBD07F536355FF477BF0E98314F667B0CA50CC7709A220B0561B85E53A48461FA8";
+
+    const wallet = Wallet.fromSeed(seed);
+
+    const signedTx: SignedTransaction = sign(wallet, tx3);
+
+    assert.equal(signedTx.signedTransaction, signedTxBlob);
+  });
+
+  it("multisigns successfully", function () {
+    const tx1: Transaction = {
+      Account: "rEuLyBCvcw4CFmzv8RepSiAoNgF8tTGJQC",
+      Fee: "30000",
+      Flags: 262144,
+      LimitAmount: {
+        currency: "USD",
+        issuer: "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh",
+        value: "100",
+      },
+      Sequence: 2,
+      Signers: [
+        {
+          Signer: {
+            Account: "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+            SigningPubKey:
+              "02B3EC4E5DD96029A647CFA20DA07FE1F85296505552CCAC114087E66B46BD77DF",
+            TxnSignature:
+              "30450221009C195DBBF7967E223D8626CA19CF02073667F2B22E206727BFE848FF42BEAC8A022048C323B0BED19A988BDBEFA974B6DE8AA9DCAE250AA82BBD1221787032A864E5",
+          },
+        },
+      ],
+      SigningPubKey: "",
+      TransactionType: "TrustSet",
+    };
+
+    const tx2: Transaction = {
+      Account: "rEuLyBCvcw4CFmzv8RepSiAoNgF8tTGJQC",
+      Fee: "30000",
+      Flags: 262144,
+      LimitAmount: {
+        currency: "USD",
+        issuer: "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh",
+        value: "100",
+      },
+      Sequence: 2,
+      Signers: [
+        {
+          Signer: {
+            Account: "rUpy3eEg8rqjqfUoLeBnZkscbKbFsKXC3v",
+            TxnSignature:
+              "30440220680BBD745004E9CFB6B13A137F505FB92298AD309071D16C7B982825188FD1AE022004200B1F7E4A6A84BB0E4FC09E1E3BA2B66EBD32F0E6D121A34BA3B04AD99BC1",
+            SigningPubKey:
+              "028FFB276505F9AC3F57E8D5242B386A597EF6C40A7999F37F1948636FD484E25B",
+          },
+        },
+      ],
+      SigningPubKey: "",
+      TransactionType: "TrustSet",
+    };
+
+    const transactions: Transaction[] = [tx1, tx2];
+
+    multisign(transactions);
+  });
+
+  it("throws a validation error if multisigning with no transactions", function () {
+    const transactions: Transaction[] = [];
+    assert.throws(() => multisign(transactions), ValidationError);
+  });
+
```

Comment:
Add additional tests for multisign - 
* When the transactions aren't identical
* With tx blobs instead of Transaction objects